### ISSUE TITLE
Vaginal birth after Caesarian HP:0030365 is a subclass of Abnormal delivery

### ISSUE THREAD

**Issue by paola-scibite** [obophenotype/human-phenotype-ontology#10908]
**HPO term**
Vaginal birth after Caesarian HP:0030365

**Suggested revision and reasons**

Vaginal birth after Caesarian "Vaginal birth after Caesarian (VBAC) refers to the situation where the mother has had a previous Cesarean delivery but has now delivered vaginally."
is currently a subclass of 
Abnormal delivery HP:0001787 "An abnormality of the birth process."
It has a Disease Association with [ORPHA:60041](https://hpo.jax.org/browse/disease/ORPHA:60041) Congenital heart block.
I suppose that might be the reason for wanting to record VBAC in HP somehow, but strictly speaking, VBAC is not an abnormal birth process...

On a minor note, "Caesarian" seems an accepted spelling for the more common "Caesarean" and "Cesarean", so I'm not complaining ;-) but I noticed an inconsistency here:
Secondary Caesarian section HP:0030364 "Delivery by Caesarian section representing where the mother has already had a previous Cesarean delivery, and this is a repeat Cesarean birth."

Thanks,

Paola

**Comment by pnrobinson** [comment]
I agree that Caesarean section is the correct spelling, I have fixed this.

It seems unlikely that Congenital heart block is truly associated with this, maybe it was a chance finding?

We are in the process of making our representation of things like this more flexible with the Phenopacket Schema, but for now I think we need to leave the term here because it will take a lot of work to transition the software.
